src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -0,0 +1,26 @@
+package juja.sqlcmd;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private static final String PORT = "5432";
+    private static final  String SERVER = "localhost";
+    private Connection connection;
+
+    public boolean connect(String database, String user, String password) {
+        String dbConnectionUrl = "jdbc:postgresql://" + SERVER + ":" + PORT + "/" + database;
+        try {
+            Class.forName("org.postgresql.Driver");
+            connection = DriverManager.getConnection(dbConnectionUrl, user, password);
+            return true;
+        } catch (ClassNotFoundException e) {
+            System.err.println("The driver was not found.");
```

Comment:
There's no requirement in task to print anything to console (http://bit.ly/2jRFHEn)

Reply:
I changed this method